lib/screens/HomeScreen.tsx

```diff
@@ -291,6 +309,11 @@ const HomeScreen: React.FC<HomeScreenProps> = ({ navigation, route }) => {
       color: theme.logoutText,
       fontWeight: "700",
     },
+    buttonText:{
+      fontSize: 16,
+      color: theme.text,
+      fontWeight: "600",
+    },
```

Comment:
Good job on the styles, they look proper and keep it concise, along with adding comments to describe what the styling is doing.